handlers/oauthhandler/oauthhandler.go

```diff
@@ -0,0 +1,182 @@
+package oauthhandler
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/google/uuid"
+	"github.com/vocdoni/blind-csp/model"
+	"github.com/vocdoni/blind-csp/types"
+	"go.vocdoni.io/dvote/log"
+)
+
+// OauthHandler is a handler that requires a verifiable oAuth token to be resolved.
+type OauthHandler struct{}
+
+// Init does nothing
+func (dh *OauthHandler) Init(opts ...string) error {
+	return nil
+}
+
+// GetName returns the name of the handler
+func (dh *OauthHandler) Name() string {
+	return "oAuth"
+}
+
+// Info returns the handler options and required auth steps.
+func (dh *OauthHandler) Info() *types.Message {
+	return &types.Message{
+		Title:    "oAuth handler",
+		AuthType: "auth",
+		SignType: types.AllSignatures,
+		AuthSteps: []*types.AuthField{
+			{Title: "GetAuthUrl", Type: "text"},
+			{Title: "VerifyElection", Type: "text"},
+		},
+	}
+}
+
+// Indexer takes a unique user identifier and returns the list of processIDs where
+// the user is elegible for participation. This is a helper function that might not
+// be implemented (depends on the handler use case).
+func (ih *OauthHandler) Indexer(userID types.HexBytes) []types.Election {
```

Comment:
`dh` is previously used as receiver name and here is been using `ih`, is a good practice to use the same for the all struct methods, but I think maybe a accurate receiver name could be `o`